src/app/components/ContactPlaceholder.js

```diff
@@ -217,27 +215,20 @@ const ContactPlaceholder = ({
             </div>
             {hasPaidMail ? (
                 <PaidCards
-                    user={user}
                     userKeysList={userKeysList}
                     loadingUserKeys={loadingUserKeys}
                     onImport={onImport}
                     onExport={() => onExport()}
                     onGroups={onGroups}
                 />
             ) : (
-                <FreeCards
-                    history={history}
-                    loadingUserKeys={loadingUserKeys}
-                    onImport={onImport}
-                    onExport={() => onExport()}
-                />
+                <FreeCards loadingUserKeys={loadingUserKeys} onImport={onImport} onExport={() => onExport()} />
```

Comment:
`onExport={onExport}` doesn't work?

Reply:
At some point it was giving me trouble for some strange reason. But indeed it should be ok. I tested it now and there is no problem.

Reply:
@econdepe Do you confirm `onExport` doesn't care on the argument when it's called?

Reply:
Yeah, I don't know why that happened to me in the first place. Maybe I was doing something crazy on that day. But it seems ok now. Both buttons (sidebar and placeholder) work as they should

Reply:
The () => onExport() is needed. That's because the function is supposed to take a parameter sometimes: `contactGroupID`. And it's necessary to specify that it is undefined in this case